### PR TITLE
Require Java 11 for plexus-compiler-eclipse an javac-errorprone and aspectj compilers

### PR DESCRIPTION
**Pull request by slachiewicz**

### CODE REVIEW

File: plexus-compiler-its/pom.xml [hunk 1]
```diff
@@ -12,9 +12,11 @@
   <packaging>pom</packaging>
 
   <name>Plexus Compiler It Tests</name>
-  
+
   <properties>
     <junit.version>4.13.2</junit.version>
+    <maven.compiler.source>11</maven.compiler.source>
```
Comment (olamy) [inline plexus-compiler-its/pom.xml:18]: why not using `release`?

**Review by olamy — COMMENTED** [review]
maybe you could wait for review before merging??

File: plexus-compiler-its/pom.xml (hunk 1 above)
Comment (mbien) [reply plexus-compiler-its/pom.xml:18]: netbeans would show this as warning, assuming it can figure out the compiler plugin version :)

Comment (slachiewicz) [reply plexus-compiler-its/pom.xml:18]: PR #277
Feel free to open or with better solution

Comment (slachiewicz) [reply plexus-compiler-its/pom.xml:18]: Feel free to improve opening pr

**Pull request merged**